src/main/kotlin/apply/ui/admin/mail/MailForm.kt

```diff
@@ -36,17 +38,16 @@ class MailForm(
     private val subject: TextField = TextField("제목").apply { setWidthFull() }
     private val body: TextArea = createBody()
     private val mailTargets: MutableSet<MailTargetResponse> = mutableSetOf()
+    private val uploadFile: MutableMap<String, ByteArrayResource> = mutableMapOf()
     private val mailTargetsGrid: Grid<MailTargetResponse> = createMailTargetsGrid(mailTargets)
     private val recipientFilter: Component = createRecipientFilter()
-    private val fileUpload: Component = createFileUpload()
-
+    private val fileUpload: Upload = createFileUpload()
     init {
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
    private val fileUpload: Upload = createFileUpload()

    init {
```